File: Templates/Bellatrix.Web.NUnit.Tests/LoginTests.cs
using System;
using NUnit.Framework;

namespace Bellatrix.Web.NUnit.Tests
{
    [TestFixture]
    [Browser(BrowserType.Chrome, BrowserBehavior.ReuseIfStarted)]
    [VideoRecording(VideoRecordingMode.DoNotRecord)]
    [ScreenshotOnFail(true)]
    public class LoginTests : WebTest
    {
        public override void TestInit() => App.NavigationService.Navigate("http://demos.bellatrix.solutions/my-account/");

        [Test]
        ////[VideoRecording(VideoRecordingMode.Ignore)]
        ////[ScreenshotOnFail(false)]
        public void SuccessfullyLoginToMyAccount()
        {
            TextField userNameField = App.ElementCreateService.CreateById<TextField>("username");
            Password passwordField = App.ElementCreateService.CreateById<Password>("password");
            Button loginButton = App.ElementCreateService.CreateByXpath<Button>("//button[@name='login']");

            userNameField.SetText("info@berlinspaceflowers.com");
            passwordField.SetPassword("@purISQzt%%DYBnLCIhaoG6$");
            loginButton.Click();

            Div myAccountContentDiv = App.ElementCreateService.CreateByClass<Div>("woocommerce-MyAccount-content");
            myAccountContentDiv.EnsureInnerTextContains("Hello info1");

            Anchor logoutLink = App.ElementCreateService.CreateByInnerTextContaining<Anchor>("Log out");

            logoutLink.EnsureIsVisible();
            logoutLink.Click();
        }

        [Test]
        public void SuccessfullyLoginToMyAccount1()
        {
            TextField userNameField = App.ElementCreateService.CreateById<TextField>("username");
            Password passwordField = App.ElementCreateService.CreateById<Password>("password");
            Button loginButton = App.ElementCreateService.CreateByXpath<Button>("//button[@name='login']");

            userNameField.SetText("info@berlinspaceflowers.com");
            passwordField.SetPassword("@purISQzt%%DYBnLCIhaoG6$");
            loginButton.Click();

            Div myAccountContentDiv = App.ElementCreateService.CreateByClass<Div>("woocommerce-MyAccount-content");
            myAccountContentDiv.EnsureInnerTextContains("Hello info1");

            Anchor logoutLink = App.ElementCreateService.CreateByInnerTextContaining<Anchor>("Log out");

            logoutLink.EnsureIsVisible();
            logoutLink.Click();
        }
    }
}

File: Templates/Bellatrix.Web.NUnit.Tests/LoginTests.vb

Namespace Bellatrix.Web.NUnit.Tests
    <NUnit.Framework.TestFixtureAttribute>
    <Browser(BrowserType.Chrome, BrowserBehavior.ReuseIfStarted)>
    <VideoRecording(VideoRecordingMode.DoNotRecord)>
    <ScreenshotOnFail(True)>
    Public Class LoginTests
        Inherits WebTest

        Public Overrides Sub TestInit()
            App.NavigationService.Navigate("http://demos.bellatrix.solutions/my-account/")
        End Sub

        <NUnit.Framework.TestAttribute>
        '''[VideoRecording(VideoRecordingMode.Ignore)]
        '''[ScreenshotOnFail(false)]
        Public Sub SuccessfullyLoginToMyAccount()
            Dim userNameField = App.ElementCreateService.CreateById(Of TextField)("username")
            Dim passwordField = App.ElementCreateService.CreateById(Of Password)("password")
            Dim loginButton = App.ElementCreateService.CreateByXpath(Of Button)("//button[@name='login']")
            userNameField.SetText("info@berlinspaceflowers.com")
            passwordField.SetPassword("@purISQzt%%DYBnLCIhaoG6$")
            loginButton.Click()
            Dim myAccountContentDiv = App.ElementCreateService.CreateByClass(Of Div)("woocommerce-MyAccount-content")
            myAccountContentDiv.ValidateInnerTextContains("Hello info1")
            Dim logoutLink = App.ElementCreateService.CreateByInnerTextContaining(Of Anchor)("Log out")
            logoutLink.ValidateIsVisible()
            logoutLink.Click()
        End Sub

        <NUnit.Framework.TestAttribute>
        Public Sub SuccessfullyLoginToMyAccount1()
            Dim userNameField = App.ElementCreateService.CreateById(Of TextField)("username")
            Dim passwordField = App.ElementCreateService.CreateById(Of Password)("password")
            Dim loginButton = App.ElementCreateService.CreateByXpath(Of Button)("//button[@name='login']")
            userNameField.SetText("info@berlinspaceflowers.com")
            passwordField.SetPassword("@purISQzt%%DYBnLCIhaoG6$")
            loginButton.Click()
            Dim myAccountContentDiv = App.ElementCreateService.CreateByClass(Of Div)("woocommerce-MyAccount-content")
            myAccountContentDiv.ValidateInnerTextContains("Hello info1")
            Dim logoutLink = App.ElementCreateService.CreateByInnerTextContaining(Of Anchor)("Log out")
            logoutLink.ValidateIsVisible()
            logoutLink.Click()
        End Sub
    End Class
End Namespace
